libs/api/src/lib/graphql/mutation.public.ts

```diff
@@ -968,10 +981,11 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
         return await createPaymentWithProvider({
           paymentMethodID: paymentMethod.id,
           invoice,
-          saveCustomer: false,
+          saveCustomer: true,
```

Comment:
Why was this changed?

Reply:
So that the new credit card details would be stored